docs/Resonance_Compensation.md

```diff
@@ -79,14 +72,14 @@ First, measure the **ringing frequency**.
 
     |![Mark ringing](img/ringing-mark.jpg)|![Measure ringing](img/ringing-measure.jpg)|
 
-10. Count how many oscillations *N* the measured distance *D* corresponds to.
-    If you are unsure how to count the oscillations, refer to the picture
-    above, which shows *N* = 6 oscillations.
-11. Compute the ringing frequency of X axis as *V* &middot; *N* / *D* (Hz),
-    where *V* is the velocity for outer perimeters (mm/sec). For the example
-    above, we marked 6 oscillations, and the test was printed at 100 mm/sec
-    velocity, so the frequency is 100 * 6 / 12.14 ≈ 49.4 Hz.
-12. Do (9) - (11) for Y mark as well.
+8. Count how many oscillations *N* the measured distance *D* corresponds to.
+   If you are unsure how to count the oscillations, refer to the picture
+   above, which shows *N* = 6 oscillations.
+9. Compute the ringing frequency of X axis as *V* &middot; *N* / *D* (Hz),
+   where *V* is the velocity for outer perimeters (mm/sec). For the example
+   above, we marked 6 oscillations, and the test was printed at 100 mm/sec
+   velocity, so the frequency is 100 * 6 / 12.14 ≈ 49.4 Hz.
+10. Do (9) - (11) for Y mark as well.
```

Comment:
This should read 'Do (7) - (9) for Y mark as well.' now (or different numbers, if the square_corner_velocity point is kept).